.github/workflows/docker.yaml

```diff
@@ -67,6 +67,7 @@ jobs:
           push: true
           build-args: |
             GIT_VERSION=${{ github.ref_type == 'tag' && github.ref_name || github.event.pull_request.head.sha || github.sha }}
+            version=${{ github.ref_type == 'tag' && github.ref_name || github.sha }}
```

Comment:
the `Dockerfile` also needs to be updated so that it accepts version.


i'd recommend using `GIT_VERSION` already defined in workflow (and the dockerfile as an ARG)

Reply:
Ok, Updated. PTAL